kommunicate-script-example/README.md

```diff
@@ -42,7 +42,54 @@ Example:
         })(document, window.kommunicate || {});
 
 ```
-> **Note:** Replace the `<APP_ID>` and `<CONVERSATION_TITLE>` with the `appId` and `conversationTitle` copied from the <a href="https://dashboard.kommunicate.io/settings/install" target="_blank">Install Section</a> from the Kommunicate Dashboard.
+> **Note:** Replace the `<APP_ID>` with the `appId` copied from the <a href="https://dashboard.kommunicate.io/settings/install" target="_blank">Install Section</a> from the Kommunicate Dashboard.
+
+
+### Make chat widget fullscreen on pageload
+To make the Kommunnicate chat widget Full screen on page-load copy & paste the following code inside the `onInit: function() { ... }` of the `kommunicateSettings` variable.
+
+> **Note:** Replace the `<APP_ID>` with the `appId` copied from the <a href="https://dashboard.kommunicate.io/settings/install" target="_blank">Install Section</a> from the Kommunicate Dashboard.
+
+Example:
+```javascript
+	(function (d, m) {
+            var kommunicateSettings = {
+                "appId": "<APP_ID>",
+                "onInit": function () {
+                    var iframeStyle = document.createElement('style');
+                    var classSettings = ".change-kommunicate-iframe-height{height:100%!important;width:100%!important;right:0!important;bottom:0!important;max-height: 100%!important;}";
+                    iframeStyle.type = 'text/css';
+                    iframeStyle.innerHTML = classSettings;
+                    document.getElementsByTagName('head')[0].appendChild(iframeStyle);
+                    var launcherIconStyle = "@media(min-width: 510px){.mck-sidebox.fade.in,.mck-box .mck-box-sm{width:100%; height:100%;max-height:100%!important;border-radius:0px!important;}.mck-sidebox{right:0!important;bottom:0!important;}}";
+                    Kommunicate.customizeWidgetCss(launcherIconStyle);
+
+                    var iframe = parent.document.getElementById("kommunicate-widget-iframe"),
+                        launcher = KommunicateGlobal.document.getElementById('mck-sidebox-launcher'),
+                        preChatPopup = KommunicateGlobal.document.querySelector('#chat-popup-widget-container .chat-popup-widget-text-wrapper'),
+                        closeButton = KommunicateGlobal.document.getElementById('km-chat-widget-close-button');
+
+                    [launcher, preChatPopup].forEach(function (element) {
+                        element.addEventListener('click', function () {
+                            iframe.classList.add("change-kommunicate-iframe-height");
+                        });
+                    });
+
+                    closeButton.addEventListener('click', function () {
+                        iframe.classList.remove("change-kommunicate-iframe-height");
+                    });
+                    Kommunicate.launchConversation();
+                    KommunicateGlobal.document.getElementById('mck-sidebox-launcher').click()
```

Comment:
please check if  Kommunicate.launchConversation(); is required if we are already using
                    KommunicateGlobal.document.getElementById('mck-sidebox-launcher').click()
are both lines of code required to make it work?

Reply:
Without that line, it's not opening as fullscreen. Just showing as a button and once clicked, then it's showing full-screen